### DISCUSSION TITLE
i installed c++ tools but still its not working...after installing the tools what should I do ?

### DISCUSSION THREAD

**Discussion by krusham** [explosion/spaCy#11487] [Help: Installation]
i installed c++ tools but still its not working...after installing the tools what should I do ?
Please help me out if you could.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> cymem
thank you

_Originally posted by @krusham in https://github.com/explosion/spaCy/discussions/10804#discussioncomment-3627016_

**Comment by adrianeboyd** [comment]
Let me convert this to a new discussion thread in the "Installation" topic.

Please edit the top message to provide more details about the install commands with details about which versions of spacy and other packages you're trying to install, and provide more details from the error trace.

**Reply by krusham** [reply]
I am using visual studio code python version 3.10
I am new to programming language and while running a project I am facing this issue 

cannot import name 'hash_unicode' from 'murmurhash'


error: legacy-install-failure

× Encountered error while trying to install package.
╰─> cymem


note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.

**Reply by adrianeboyd** [reply]
We can't really help given the info provided so far, so please provide more details about what code you're trying to run and what you're trying to do.

This looks like an error message from `pip install`. What command did you run or how are you trying to install spacy?

**Reply by krusham** [reply]
installed spacy through pip install spacy 
pip install ..\en_core_web_sm-1.2.0\

**Reply by krusham** [reply]
i am running code for searching keywords I installed chrome driver visual c++ tools but still its showing error

**Reply by adrianeboyd** [reply]
Are you sure that you definitely need to use `en_core_web_sm-1.2.0`? This is an old version of the model that requires an old version of spacy, where both were published more than five years ago. All the required packages were developed years before python 3.10 was released, so they were only tested with older versions of python and there are various incompatibilities.

If you can, I would recommend trying to find a newer project as a starting point, in particular one that has been updated for spacy v3, since there have been a lot of improvements in spacy and many things will be easier.

If you need to use this exact version of the model, then you might have an easier time with python 3.6, which was the newest version of python at the time of this release. Also note that python 3.6 has already reached its end of life and is no longer updated.
